doc-Infrastructure_Migration_Solution_Guide/cfme/modules/proc_Installing_configuring_multi-cf.adoc

```diff
@@ -0,0 +1,67 @@
+// Module included in the following assemblies:
+//
+// IMS_1.2/master.adoc
+// IMS_1.3/master.adoc
+[id="Installing_configuring_cf_distributed_{context}"]
+= Installing and configuring Red Hat CloudForms in a distributed environment
+
+You can install Red Hat CloudForms in a distributed environment, with CloudForms appliances on your user interface, database, and worker machines.
+
+[NOTE]
+====
+If you create an infrastructure mapping or migration plan and then remove or change a provider in CloudForms, the mapping or plan must be deleted and re-created.
+====
+
+.Procedure
+
+ifdef::rhv_1-2_vddk,rhv_1-3_vddk[]
+. Install Red Hat CloudForms on link:https://access.redhat.com/documentation/en-us/red_hat_cloudforms/5.0/html/installing_red_hat_cloudforms_on_red_hat_virtualization[Red Hat Virtualization (RHV)] or link:https://access.redhat.com/documentation/en-us/red_hat_cloudforms/5.0/html/installing_red_hat_cloudforms_on_vmware_vsphere/index[VMware].
+endif::[]
+ifdef::osp_1-2_vddk,osp_1-3_vddk[]
+. Install Red Hat CloudForms on link:https://access.redhat.com/documentation/en-us/red_hat_cloudforms/5.0/html-single/installing_red_hat_cloudforms_on_red_hat_openstack_platform/[Red Hat OpenStack Platform (RHOSP)] or link:https://access.redhat.com/documentation/en-us/red_hat_cloudforms/5.0/html/installing_red_hat_cloudforms_on_vmware_vsphere/index[VMware].
+endif::[]
+
+. link:https://access.redhat.com/documentation/en-us/red_hat_cloudforms/5.0/html-single/general_configuration/index#servers[Enable the required server roles] in the CloudForms user interface:
+
+* Database: No server roles required
+* User interface:
+
+**  Database Operations
+**  User Interface
+**  Web Services
+**  Websocket
```

Comment:
I could not find any "Websocket" option in server roles, I have checked it in  CFME version 5.11.4.2 and also in upstream appliance ManageIQ Kasparov, Could you please have a look into it, I have attached screenshot for the same.

Reply:
You are correct. Although it's still in the documentation (https://access.redhat.com/documentation/en-us/red_hat_cloudforms/5.0/html-single/general_configuration/index#region-aware-server-roles), I could not find it in the UI. Investigating